resources/resources/software/aws/toolkits/resources/MessagesBundle.properties

```diff
@@ -89,6 +89,9 @@ aws.settings.aws_cli_settings=AWS CLI Settings
 aws.settings.clouddebug.location=cloud-debug CLI executable:
 aws.settings.dynamic_resources_configurable.clear_all=Clear All
 aws.settings.dynamic_resources_configurable.select_all=Select All
+aws.settings.dynamic_resources_configurable.suggest_types.dialog.message=Please suggest additional AWS resource types you would like to see (e.g. AWS::S3::Bucket).\n(max length: 2000 chars)
```

Comment:
Could we say "like to see in the future"?

Reply:
Do you think that makes it clearer? maybe "like to see added in future releases?"

Reply:
I think "suggest"  makes it clear that the intent is for the future, but just to be more explicit, we could probably add those words?

Reply:
Could just say 'like to see added' if you want to be concise.